### ISSUE TITLE
Suggestion: Replace UNetbootin with Startup Disk Creator

### ISSUE THREAD

**Issue by tpow** [elementary/website#2173]
I thought the installation process for Elementary would be as elegant as the OS itself, but UNetbootin was a pain. I got partway through setting it up and thought there must be a better way to create the bootable USB.

I found that if you're already on Ubuntu, the easiest is the Startup Disk Creator tool. I used it right after downloading the ISO and it pre-populated everything for me and worked wonderfully.

Here's an article that describes the process:
https://linuxconfig.org/how-to-create-a-bootable-ubuntu-18-04-bionic-usb-stick-on-linux 

The only difference with elementary OS is the name of the ISO.

I was using Ubuntu 18.04, but this article indicates that Startup Disk Creator works with 16.04 as well.

I would suggest changing the installation instructions to use the already installed Startup Disk Creator tool.

**Comment by peteruithoven** [comment]
See similar discussions to use Etcher (for all platforms): https://github.com/elementary/website/issues/1489
I think one interesting question would be whether Startup Disk Creator is available for other distro's?

**Comment by NathanBnm** [comment]
Personnaly I use [Rufus ](https://rufus.ie) on Windows 10 and it's working well 👍

**Comment by felipeozalmeida** [comment]
First of all, I want to congrats all the guys from elementary team! I've bought the OS and I went to the installation instructions. I've already installed elementary before using another tools, but now I just wanted to follow the docs. Unfortunately, I didn't make it with UNetbootin: it doesn't even showed me a GUI after all on a fresh Ubuntu 18.04.2! I've installed UNetbootin from their PPA as they recommend and I don't understand why it doesn't work. I'm following with a more modern tool now (Etcher) and I think the installation instructions for Linux could be updated. Thank you all!

![Screenshot from 2019-07-27 02-52-59](https://user-images.githubusercontent.com/30023777/61990591-b8a4f480-b019-11e9-91b7-0eed0b0f221c.png)
